docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
   node:
     image: aeternity/aeternity:${TAG}
     hostname: node
-    expose: [3013, 3113]
+    ports: ["3013:3013", "3113:3113"]
```

Comment:
this should be kept as expose not ports. the docker file is used by the ci where potentially parallel jobs are executed on the same host.

Reply:
Is it still possible to expose it to the host via docker-compose then? I'd like to run this instead of the aeternity 3 node testnet